src/main/resources/edu/wpi/first/pathweaver/createProject.fxml

```diff
@@ -48,13 +48,13 @@
             <TextField fx:id="maxAcceleration" GridPane.columnIndex="1" GridPane.rowIndex="3" />
             <TextField fx:id="maxJerk" GridPane.columnIndex="1" GridPane.rowIndex="4" />
             <TextField fx:id="wheelBase" GridPane.columnIndex="1" GridPane.rowIndex="5" />
-            <Button fx:id="browse" mnemonicParsing="false" text="Browse" GridPane.columnIndex="2" />
+            <Button fx:id="browse" mnemonicParsing="false" text="Browse" onAction="#browseDirectory" GridPane.columnIndex="2" />
             <Label text="Time delta between points (seconds)" GridPane.columnIndex="2" GridPane.rowIndex="1" />
             <Label text="Maximum capable velocity of robot" GridPane.columnIndex="2" GridPane.rowIndex="2" />
             <Label text="Maximum acceleration (velocity per second)" GridPane.columnIndex="2" GridPane.rowIndex="3" />
             <Label text="Maximum jerk (acceleration per second)" GridPane.columnIndex="2" GridPane.rowIndex="4" />
             <Label text="Distance between left and right of wheel base" GridPane.columnIndex="2" GridPane.rowIndex="5" />
-            <Button fx:id="create" mnemonicParsing="false" text="Create Project" GridPane.columnIndex="2" GridPane.rowIndex="6" />
+            <Button fx:id="create" mnemonicParsing="false" text="Create Project" onAction="#createProject" GridPane.columnIndex="2" GridPane.rowIndex="6" />
```

Comment:
There should be another button for going back to the welcome screen. Right now, you'll have to restart the app to open an existing project if the user decides not to create a new project or accidentally pressed the button